windows/ReactNativeWebView/pch.h

```diff
@@ -14,6 +14,7 @@
 #include <winrt/Windows.Web.Http.Headers.h>
 #include <winrt/Microsoft.ReactNative.h>
 #include <winrt/Microsoft.UI.Xaml.Controls.h>
+#include <winrt/Microsoft.Web.WebView2.Core.h>
```

Comment:
this will only be present if using a WinUI 2.x version with WV2 (prerelease only as of today), so it needs to be guarded so that folks can use react-native-webview to use WV1 if they're using WinUI 2.6, 2.7, etc.